tasks.py

```diff
@@ -592,6 +592,14 @@ def pylint(context, target=None, recursive=False):
         run_command(context, command)
 
 
+@task
+def docs_local(context):
+    """Runs local instance of mkdocs serve (ctrl-c to stop)."""
+    command = "poetry run mkdocs serve"
```

Comment:
I don't think I agree with prepending `poetry run` to our invoke tasks - in general you should always use `poetry shell` before running `invoke`. Otherwise we'd need `poetry run nautobot-server ....`, `poetry run flake8`, `poetry run black`, ...